examples/lib/pods_test.rego

```diff
@@ -0,0 +1,91 @@
+package lib.pods
+
+test_input_as_other {
+    input := {
+        "kind": "Other",
+        "spec": {
+            "containers": [{}]
+        }
+    }
+
+    resource := pod with input as input
+
```

Comment:
nit: This empty line doesn't seem necessary to me for readability. I think this line should be removed throughout the tests.

Reply:
I'm not incredibly opinionated either way, but this is _typically_ how I organize my tests across all languages. It draws from [Arrange Act Assert (AAA)](http://wiki.c2.com/?ArrangeActAssert) -- where 

- `input` is getting the test set up
- `resource := pod ..` calls the pod value / acts
- `not resource` asserts our expectation

Reply:
:+1: Due to the input being assigned to a multi-line object with varying levels of indentation, this doesn't have the same impact for me as in your linked example, but this makes sense.